### PR TITLE
Feature/apl int test 2 precondition

### PR DESCRIPTION
**Pull request by maksimkhabenko**

### CODE REVIEW

File: apl-integration-tests/src/test/java/com/apollocurrrency/aplwallet/inttest/model/TestBase.java [hunk 1]
```diff
@@ -1,9 +1,6 @@
 package com.apollocurrrency.aplwallet.inttest.model;
 
-import com.apollocurrency.aplwallet.api.dto.AccountDTO;
-import com.apollocurrency.aplwallet.api.dto.BalanceDTO;
-import com.apollocurrency.aplwallet.api.dto.ForgingDetails;
-import com.apollocurrency.aplwallet.api.dto.TransactionDTO;
+import com.apollocurrency.aplwallet.api.dto.*;
```
Comment (AndrewBoyarsky) [inline apl-integration-tests/src/test/java/com/apollocurrrency/aplwallet/inttest/model/TestBase.java]: Please review all code to disallow wildcard imports

File: apl-integration-tests/src/test/java/com/apollocurrrency/aplwallet/inttest/model/TestBase.java [hunk 2]
```diff
@@ -48,9 +46,15 @@ static void initAll() {
...
         String secretFilePath = Objects.requireNonNull(classLoader.getResource("APL-MK35-9X23-YQ5E-8QBKH")).getPath();
-        importSecretFileSetUp(secretFilePath,"1");
-        startForgingSetUp();
-        setUpTestData();
+        try {
+            importSecretFileSetUp(secretFilePath,"1");
+            startForgingSetUp();
+            setUpTestData();
+        }catch (Exception ex){
+            Assert.assertNotNull("Precondition FAILED", ex);
```
Comment (AndrewBoyarsky) [inline apl-integration-tests/src/test/java/com/apollocurrrency/aplwallet/inttest/model/TestBase.java]: Looks like such statement makes no sense

File: apl-integration-tests/src/test/java/com/apollocurrrency/aplwallet/inttest/tests/TestSharding.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        List<ShardDTO> maxShardsList = new ArrayList<>();
+        int maxHeight = 0;
+
+        for (String ip: peers) {
+            try {
+                List<ShardDTO> shardDTOS = getShards(ip);
+                if (shardDTOS.size() > maxShardsList.size()){maxShardsList =shardDTOS;}
+                shards.put(ip, shardDTOS);
+                heights.put(ip,getLastBlock(ip).getHeight());
+                System.out.println(getLastBlock(ip));
```
Comment (AndrewBoyarsky) [inline apl-integration-tests/src/test/java/com/apollocurrrency/aplwallet/inttest/tests/TestSharding.java]: Maybe use logger?

File: apl-integration-tests/src/test/java/com/apollocurrrency/aplwallet/inttest/model/TestBase.java (hunk 2 above)
Comment (maksimkhabenko) [reply apl-integration-tests/src/test/java/com/apollocurrrency/aplwallet/inttest/model/TestBase.java]: This is done specifically for display in the report.

File: apl-integration-tests/src/test/java/com/apollocurrrency/aplwallet/inttest/tests/TestSharding.java (hunk 3 above)
Comment (maksimkhabenko) [reply apl-integration-tests/src/test/java/com/apollocurrrency/aplwallet/inttest/tests/TestSharding.java]: Yes, it is temporary code. I will be delete it.

File: apl-integration-tests/src/test/java/com/apollocurrrency/aplwallet/inttest/model/NetConfig.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+import java.util.ArrayList;
+import java.util.List;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString
+public class NetConfig {
```
[the remark this answers is not in the document]
Comment (maksimkhabenko) [reply apl-integration-tests/src/test/java/com/apollocurrrency/aplwallet/inttest/model/NetConfig.java]: Done

File: apl-integration-tests/src/test/java/com/apollocurrrency/aplwallet/inttest/model/NetConfig.java [hunk 5]
```diff
@@ -0,0 +1,16 @@
+package com.apollocurrrency.aplwallet.inttest.model;
+
+import lombok.*;
```
[the remark this answers is not in the document]
Comment (maksimkhabenko) [reply apl-integration-tests/src/test/java/com/apollocurrrency/aplwallet/inttest/model/NetConfig.java]: Done

**Pull request merged**